pkg/virt-handler/node-labeller/node_labeller.go

```diff
@@ -166,6 +179,35 @@ func (n *NodeLabeller) run() error {
 	return err
 }
 
+func (n *NodeLabeller) getKVMInfoLabels() (map[string]bool, error) {
+	labels := make(map[string]bool)
+	filestat, err := os.Stat(kvmCapsInfoPluginPath)
+	if err != nil {
+		return labels, err
+	}
+
+	//if file is regular and executable
+	if filestat.Mode().IsRegular() && filestat.Mode()&0111 != 0 {
+		cmd := exec.Command(kvmCapsInfoPluginPath)
+		var stdout, stderr bytes.Buffer
+		cmd.Stdout = &stdout
+		cmd.Stderr = &stderr
+
+		// Run hook
+		err = cmd.Run()
```

Comment:
what happens if cmd.Run() never completes. I think it would be wise to have a timeout here of some sort.